src/bridge.ts

```diff
@@ -1165,13 +1164,14 @@ export class Bridge {
                 )
             },
             ...this.opts.intentOptions?.clients,
-            onEventSent: () => this.opts.controller.userActivityTracker?.updateUserActivity(userId!),
+            onEventSent: () => userId && this.opts.controller.userActivityTracker?.updateUserActivity(userId),
         };
         clientIntentOpts.registered = this.membershipCache.isUserRegistered(userId);
         const encryptionOpts = this.opts.bridgeEncryption;
         if (encryptionOpts) {
             clientIntentOpts.encryption = {
                 sessionPromise: encryptionOpts.store.getStoredSession(userId),
+                origianlHomeserver: this.opts.homeserverUrl,
```

Comment:
typo that hopefully still can be fixed in the entire repo.
```suggestion
                originalHomeserver: this.opts.homeserverUrl,
```